src/main/webapp/pages/oauth/access_confirmation.html

```diff
@@ -1,53 +1,94 @@
 <!DOCTYPE html>
 <html xmlns:th="http://www.thymeleaf.org">
-<head>
+  <head>
     <title th:text="#{auth.title}">IRIDA OAuth2 Approval Page</title>
-    <link rel="stylesheet" th:href="@{/resources/dist/css/bootstrap.bundle.css}">
+    <link
+      rel="stylesheet"
+      href="/resources/dist/css/access_confirmation.bundle.css"
+    />
     <style>
-        .button-form {
-            display: inline-block;
-        }
-
-        .modal-footer {
-            position: absolute;
-            bottom: 0;
-            left: 0;
-            right: 0;
+        body {
+            padding: 14px;
         }
+      form {
+        display: inline-block;
+      }
     </style>
-</head>
-<body>
-<section class="modal-body" style="margin-top: 0;">
-    <h3 style="margin-top: 0" th:text="#{auth.title}">_IRIDA OAuth2 Approval Page_</h3>
+  </head>
+  <body>
+    <section class="modal-body" style="margin-top: 0;">
```

Comment:
If I click **Deny** or the **X** icon in this modal

![image](https://user-images.githubusercontent.com/200517/53756597-1d9b5a80-3e7f-11e9-8541-3cabbdbbc26a.png)

Then the **Authenticating Galaxy Client** button will continue to be greyed out and appear as if it's working. The only way to reset it is to refresh the page:

![image](https://user-images.githubusercontent.com/200517/53756494-e0cf6380-3e7e-11e9-8f34-2575808f555e.png)

Reply:
Sorry @apetkau I don't know what you mean by this